static/usage/avatar/basic/react.md

```diff
@@ -0,0 +1,15 @@
+```tsx
+import React from 'react';
+import { IonAvatar } from '@ionic/react';
+
+function Example() {
+  return (
+    <>
+      <IonAvatar>
+        <img src="https://ionicframework.com/docs/demos/api/avatar/avatar.svg" />
```

Comment:
I get the following message in the console in StackBlitz:

```
[eslint] 
src/main.tsx
  Line 8:9:  img elements must have an alt prop, either with meaningful text, or an empty string for decorative images  jsx-a11y/alt-text
```

We should probably do this for all the usages of `img` in this PR.

Reply:
Resolved by https://github.com/ionic-team/ionic-docs/pull/2524/commits/1d58b6200ade2eb9b02e0afaa0e39e49604a5fb8